app/views/layouts/manage_metadata.html.erb

```diff
@@ -33,13 +33,14 @@
 
               <nav>
```

Comment:
Why don't we take this `<nav>` and make it a partial so the different layouts stop duplicating code.